docs/src/examples.md

```diff
@@ -41,7 +41,59 @@ julia> round.(predict(ols), digits=5)
  6.83333
 ```
 
-.
+## Categorical variables/dummy coding
+
+Categorical variables will be dummy coded if they are marked as categorical in an input DataFrame. Alternatively, you can pass an explicit dictionary of contrasts.
```

Comment:
Variables which are not numeric will also be treated as categorical by default. Also, dummy coding is only the default.